### ISSUE TITLE
Failed to activate the atom-ternjs package

### ISSUE THREAD

**Issue by ocoka** [tststs/atom-ternjs#150]
[Enter steps to reproduce below:]

1. Install fresh 1.0.19 atom
2. Install atom-ternjs package

**Atom Version**: 1.0.19
**System**: Ubuntu 15.04
**Thrown From**: [atom-ternjs](https://github.com/tststs/atom-ternjs) package, v0.8.3

### Stack Trace

Failed to activate the atom-ternjs package

```
At ENOENT: no such file or directory, stat 'atom://config'

Error: ENOENT: no such file or directory, stat 'atom://config'
  at Error (native)
  at fs.statSync (fs.js:854:18)
  at Object.fs.statSync (ATOM_SHELL_ASAR.js:225:16)
  at Helper.module.exports.Helper.isDirectory (/home/ocoka/.atom/packages/atom-ternjs/lib/atom-ternjs-helper.coffee:25:8)
  at Manager.module.exports.Manager.initServers (/home/ocoka/.atom/packages/atom-ternjs/lib/atom-ternjs-manager.coffee:81:36)
  at new Manager (/home/ocoka/.atom/packages/atom-ternjs/lib/atom-ternjs-manager.coffee:35:6)
  at Object.module.exports.activate (/home/ocoka/.atom/packages/atom-ternjs/lib/atom-ternjs.coffee:87:20)
  at Package.module.exports.Package.activateNow (/usr/share/atom/resources/app.asar/src/package.js:245:19)
  at /usr/share/atom/resources/app.asar/src/package.js:226:30
  at Package.module.exports.Package.measure (/usr/share/atom/resources/app.asar/src/package.js:169:15)
  at Package.module.exports.Package.activate (/usr/share/atom/resources/app.asar/src/package.js:218:14)
  at PackageManager.module.exports.PackageManager.activatePackage (/usr/share/atom/resources/app.asar/src/package-manager.js:486:21)
  at /usr/share/atom/resources/app.asar/node_modules/settings-view/lib/package-manager.js:371:29
  at exit (/usr/share/atom/resources/app.asar/node_modules/settings-view/lib/package-manager.js:60:16)
  at triggerExitCallback (/usr/share/atom/resources/app.asar/src/buffered-process.js:213:47)
  at ChildProcess.<anonymous> (/usr/share/atom/resources/app.asar/src/buffered-process.js:235:18)
  at emitTwo (events.js:87:13)
  at ChildProcess.emit (events.js:172:7)
  at Process.ChildProcess._handle.onexit (internal/child_process.js:200:12)

```

### Commands

```
     -6:45.7.0 core:backspace (atom-text-editor#editor.fontFamily.editor.mini.is-focused)
  5x -6:45.1.0 core:delete (atom-text-editor#editor.fontFamily.editor.mini.is-focused)
     -5:46.3.0 core:paste (atom-text-editor#editor.fontFamily.editor.mini.is-focused)
  2x -2:58.8.0 core:confirm (atom-text-editor.editor.mini.is-focused)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "metrics",
      "language-toml",
      "language-ruby",
      "language-ruby-on-rails",
      "language-perl",
      "language-objective-c",
      "language-csharp",
      "language-coffee-script",
      "language-clojure",
      "language-c"
    ]
  }
}
```

### Installed Packages

```coffee
# User
atom-ternjs, v0.8.3

# Dev
No dev packages
```

**Comment by RishiKulshreshtha** [comment]
Exactly the same issue!

**Comment by Zetaphor** [comment]
@ocoka How did you resolve this issue?

**Comment by tststs** [comment]
Please open another issue with an updated stack trace. This issue is way outdated.

**Comment by ocoka** [comment]
@Zetaphor Hi, I think this issue was opened automatically by Atom (atomatically ;) )

**Comment by PixelT** [comment]
This error back in 1.17.1

**Comment by brokenpeace** [comment]
```[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.19.5 x64
**Electron**: 1.6.9
**OS**: Mac OS X 10.12.6
**Thrown From**: [atom-ternjs](https://github.com/tststs/atom-ternjs) package 0.18.3


### Stack Trace

Failed to activate the atom-ternjs package

```
At Arguments to CompositeDisposable.add must have a .dispose() method

TypeError: Arguments to CompositeDisposable.add must have a .dispose() method
    at assertDisposable (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/composite-disposable.js:74:19)
    at CompositeDisposable.module.exports.CompositeDisposable.add (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/composite-disposable.js:43:17)
    at requireAutocompleteManagerAsync (/Applications/Atom.app/Contents/Resources/app/node_modules/autocomplete-plus/lib/main.js:110:29)
    at Object.requireAutocompleteManagerAsync (/Applications/Atom.app/Contents/Resources/app/node_modules/autocomplete-plus/lib/main.js:31:15)
    at Object.consumeProvider (/Applications/Atom.app/Contents/Resources/app/node_modules/autocomplete-plus/lib/main.js:107:16)
    at Object.consumeProvider_2_0 (/Applications/Atom.app/Contents/Resources/app/node_modules/autocomplete-plus/lib/main.js:86:23)
    at Provider.module.exports.Provider.provide (/Applications/Atom.app/Contents/Resources/app/node_modules/service-hub/lib/provider.js:34:58)
    at ServiceHub.module.exports.ServiceHub.provide (/Applications/Atom.app/Contents/Resources/app/node_modules/service-hub/lib/service-hub.js:30:26)
    at Package.module.exports.Package.activateServices (/Applications/Atom.app/Contents/Resources/app/src/package.js:446:77)
    at Package.module.exports.Package.activateNow (/Applications/Atom.app/Contents/Resources/app/src/package.js:256:22)
    at /Applications/Atom.app/Contents/Resources/app/src/package.js:968:31
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:25:20)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:141:34)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:246:26)
    at CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:3:65)
    at CommandRegistry.module.exports.CommandRegistry.dispatch (/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:166:25)
    at AtomEnvironment.module.exports.AtomEnvironment.dispatchApplicationMenuCommand (/Applications/Atom.app/Contents/Resources/app/src/atom-environment.js:1338:34)
    at EventEmitter.outerCallback (/Applications/Atom.app/Contents/Resources/app/src/application-delegate.js:334:31)
    at emitThree (events.js:116:13)
    at EventEmitter.emit (events.js:194:7)
```

### Commands

```
     -4:22.6.0 core:move-left (input.hidden-input)
     -4:22.1.0 core:backspace (input.hidden-input)
     -4:21.3.0 core:move-down (input.hidden-input)
     -4:21 core:move-right (input.hidden-input)
     -4:20.4.0 core:move-up (input.hidden-input)
  2x -4:19.9.0 core:move-left (input.hidden-input)
     -4:19 core:move-down (input.hidden-input)
  2x -4:18.2.0 core:backspace (input.hidden-input)
     -4:17 editor:newline (input.hidden-input)
     -4:08.8.0 core:save (input.hidden-input)
  2x -3:59.5.0 outline-view:toggle (ul.list-inline.tab-bar.inset-panel)
     -3:08.3.0 tern:start-server (ul.list-inline.tab-bar.inset-panel)
     -2:48.5.0 core:confirm (input.hidden-input)
     -2:35.8.0 core:move-left (input.hidden-input)
     -2:33.6.0 core:confirm (input.hidden-input)
     -0:58.5.0 atom-ternjs:openConfig (div.package-readme.native-key-bindings)
```

### Non-Core Packages

```
api-workbench 0.8.47 
atom-babel-compiler 2.3.1 
atom-beautify 0.30.5 
atom-compile-coffee 1.6.0 
atom-less 0.1.10 
atom-live-server 2.1.2 
atom-react-instyle 0.2.8 
atom-ternjs 0.18.3 
atom-typescript 11.0.7 
autocomplete-json 5.5.0 
autocomplete-modules 1.7.0 
autocomplete-robot-framework 3.3.0 
build 0.68.0 
build-coffee 0.8.2 
busy 0.7.0 
busy-signal 1.4.3 
code-peek 1.4.21 
coffee-autocompile 0.6.2 
coffee-compile 0.26.0 
coffee-links 0.1.0 
coffee-navigator 0.0.18 
file-icons 2.1.11 
format-graphql 0.1.2 
go-hyperclick 1.1.2 
graphql-autocomplete 1.0.4 
haskell-grammar 0.4.0 
highlight-selected 0.13.1 
hyperclick 0.1.3 
hyperclick-robot-framework 1.6.0 
intentions 1.1.5 
js-hyperclick 1.11.0 
language-awk 0.1.12 
language-babel 2.73.0 
language-expressionengine 0.2.3 
language-graphql 0.9.0 
language-graphql-lb 0.3.1 
language-ini 1.19.0 
language-js-specs 0.1.0 
language-lua 0.9.11 
language-ocaml 1.9.5 
language-ocaml-fix 1.11.0 
language-robot-framework 2.2.0 
language-rust 0.4.12 
language-swift 0.5.0 
language-thrift 1.0.2 
language-tmux 0.4.0 
linter 2.2.0 
linter-coffees6 2.0.0 
linter-coffeescript 1.0.0 
linter-ui-default 1.6.8 
MagicPython 1.0.12 
modern-react-snippets 0.3.0 
nuclide 0.250.0 
nuclide-format-js 0.0.41 
php-hyperclick 1.2.1 
react-es6-snippets 0.3.0 
react-es7-snippets 0.3.13 
react-snippets 0.9.0 
set-syntax 0.3.2 
sort-lines 0.15.0 
Tern 0.8.0 
tool-bar 1.1.0 
tool-bar-atom 1.7.0 
```
